java/number-formatting.md

```diff
@@ -0,0 +1,36 @@
+# NumberFormatting 
+
+## Assignment
+Format a number for a specific locale. For example, the number 12345.124 should be formatted as `12,345.12` in the USA locale, but `12.345,78` in the German locale.
```

Comment:
Why are the two numerical values different here? What do you think about using the same number throughout the article for consistency?